src/javascript/JContent/JContent.accordion-items.jsx

```diff
@@ -105,7 +120,7 @@ export const jContentAccordionItems = registry => {
         icon: <FolderSpecial/>,
         label: 'jcontent:label.contentManager.navigation.contentFolders',
         defaultPath: siteKey => '/sites/' + siteKey + '/contents',
-        canDisplayItem: node => contentsRegex.test(node.path),
+        canDisplayItem: (selectionNode, folderNode) => selectionNode ? contentsRegex.test(selectionNode.path) : folderRegex.test(folderNode.path),
```

Comment:
not sure about the generic folderRegex - media can display /sites/xx/files , /sites/xx/files/xxx, but not /sites/xx/contents or  /sites/xx/contents/xxx - and only pages can display /sites/xx . here folderRegex matches all of them ?

Reply:
That's true, I'll have to do a dedicated one